src/components/dialogs/cancel.vue

```diff
@@ -46,14 +46,17 @@ import { NrAction } from '@/enums'
 import { ApplicantI } from '@/interfaces'
 import { ActionBindingIF } from '@/interfaces/store-interfaces'
 import { CANCEL_MODAL_IS_VISIBLE } from '@/modules/payment/store/types'
+import NamexServices from '@/services/namex.services'
+import getters from '@/modules/payment/store/getters'
```

Comment:
What is this used for?

Reply:
oops that was left in while I was making changes - will remove